website/source/docs/platform/k8s/run.html.md

```diff
@@ -256,6 +256,32 @@ the clients, set the `client.image` value to the desired Consul version.
 Then, run `helm upgrade`. This will upgrade the clients in batches, waiting
 until the clients come up healthy before continuing.
 
+### Using Existing PVCs
+
+The only way to use a pre-created PVC is to name them in the format Kubernetes expects:
+
+```
+data-<kubernetes namespace>-<release name>-consul-server-<ordinal>
+```
+
+The Kubernetes namespace you are installing into, release name, and ordinal must match between your Consul servers and your pre-created PVCs. You only need as many PVCs as you have Consul servers. For example, given a kube namespace of "vault" and a release name of "consul" and 5 servers, you need to create PVCs with names:
+
+```
+data-vault-consul-consul-server-0
+data-vault-consul-consul-server-1
+data-vault-consul-consul-server-2
+data-vault-consul-consul-server-3
+data-vault-consul-consul-server-4
+```
+
+You can specifiy a release name for your deployment by using the `--name` flag with `helm install`. For example:
```

Comment:
This is mandatory with Helm 3 so maybe just delete this line and the one below it.